main.js

```diff
@@ -142,4 +142,13 @@ function insertResultCard (firstUserName, secondUserName, winner) {
       <span class="align-right"><span class="close-button-circle">&times;</span></span>
     </div>
   </section>`)
+  deleteResultsCardOnClick();
```

Comment:
I used git rebase master to bring my branch up to the current and that fixed the issues. Also, the deleteResultsCardOnClick doesn't add js to the html, it's placed there so the event listener is attached after the result card is inserted.

Reply:
As far as an event listener that bubbles down to each x button in the right column to be clicked on, I don't know of one. The event that bublbles down would also have to be re-applied every time a card is inserted because the new html wouldn't have any attached event listeners.

Reply:
As far as an event listener that bubbles to target only the x button to be clicked on, I don't know of one. Click bubbles, but it targets the entire parent element inward, not just the x button. The event that bubbles would also have to be re-applied every time a card is inserted because bubbling happens during the bubble phase when an event listener is being applied, in which case, the next result card wouldn't exist yet.

Reply:
I used git rebase master to bring my branch up to the current and that fixed the issues. Also, the deleteResultsCardOnClick doesn't add js to the html, it's placed there so the event listener is attached after the result card is inserted.

Reply:
As far as an event listener that bubbles to target only the x button to be clicked on, I don't know of one. Click bubbles, but it targets the entire parent element inward, not just the x button. The event that bubbles would also have to be re-applied every time a card is inserted because bubbling happens during the bubble phase when an event listener is being applied, in which case, the next result card wouldn't exist yet.